content/blog/2019/02/2019-02-22-serverless-jenkins.adoc

```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+title: "Run your Jenkins pipeline, without running a Jenkins"
```

Comment:
```suggestion
title: "Run your Jenkins pipeline, without running Jenkins"
```

Reply:
I guess "a Jenkins instance" conveys the intend meaning better. I agree that the old wording is not so clear.

Reply:
👍